napi.h

```diff
@@ -1657,7 +1657,7 @@ namespace Napi {
     static PropertyDescriptor InstanceValue(Symbol name,
                                             Napi::Value value,
                                             napi_property_attributes attributes = napi_default);
-
+    virtual void Finalize(Napi::Env env);
```

Comment:
Nit: The blank line before `private:` ought to remain.

Reply:
Got it!  Will push in next commit.